test/extensions/filters/http/grpc_json_transcoder/grpc_json_transcoder_integration_test.cc

```diff
@@ -112,9 +112,9 @@ class GrpcJsonTranscoderIntegrationTest
       response_headers.setStatus(200);
       response_headers.setContentType("application/grpc");
       if (grpc_response_messages.empty() && !always_send_trailers) {
-        response_headers.setGrpcStatus(static_cast<uint64_t>(grpc_status.error_code()));
-        response_headers.setGrpcMessage(absl::string_view(grpc_status.error_message().data(),
-                                                          grpc_status.error_message().size()));
+        response_headers.setGrpcStatus(static_cast<uint64_t>(grpc_status.code()));
+        response_headers.setGrpcMessage(
+            absl::string_view(grpc_status.message().data(), grpc_status.message().size()));
```

Comment:
Unfortunately, that does not work because `google::protobuf::util::Status::message` returns a protobuf `StringPiece`, which does not have a conversion to `absl::string_view`.

Reply:
Would grpc_status.message().ToString() work?

Reply:
Sure, that works.